spec/controllers/projects/project_follows_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Projects::ProjectFollowsController, type: :controller do
+  let(:user) { create :user }
+  let(:episode) { create :episode }
+  let(:project) { create :project }
+
+  before :each do
+    sign_in user
+  end
+
+  describe 'GET create' do
+    it 'creates a unique project_follow' do
+      expect {
+        get :create, project_id: project.id, episode: episode, id: project.id
+      }.to change(ProjectFollow, :count).by(1)
+      expect {
+        get :create, project_id: project.id, episode: episode, id: project.id
+      }.to change(ProjectFollow, :count).by(0)
+    end
+  end
+  describe 'GET destroy' do
+    it 'deletes a project_follow entry' do
+      get :create, project_id: project.id, episode: episode, id: project.id
```

Comment:
Better use a factory for that. :smile_cat:

Reply:
need some help with this :|
factory for project_follow is created but the delete request in this method doesnt destroy that project_follow entry.